apps/web/playwright/webhook.e2e.ts

```diff
@@ -580,6 +580,8 @@ test.describe("MEETING_ENDED, MEETING_STARTED", async () => {
     await page.click('[data-testid="webhook-switch"]');
 
     await page.waitForLoadState("networkidle");
+    // wait for request to complete
+    await expect(page.getByRole("button", { name: "Disabled" })).toBeVisible();
```

Comment:
Sometimes mutation would take longer than 500ms and it would consider the loadState to be networkidle and proceed to query db before request was complete.

![Screenshot 2024-08-26 210643](https://github.com/user-attachments/assets/b6c8c65b-4a83-428a-a168-a3fcf6ef2014)

I logged for onMutate and onSuccess and db was queried before mutation could succeed.